lang/ja.yml

```
@@ -83,110 +83,110 @@ INSPECTOR_TOGGLED: "検査官モードが{有効|無効}になりました。"
+LOOKUP_HEADER: "{0} の結果"
+LOOKUP_INTERACTION: "{0}が{1}を{クリック|破壊}した。"
+LOOKUP_ITEM: "{0}が{2} {1}を{拾った|落とした}。"
+LOOKUP_LOGIN: "{0}の{ログイン|ログアウト}がログに記録されます。"
+LOOKUP_PAGE: "{0} ページ目"
+LOOKUP_PROJECTILE: "{0}が{2} {1}を{投げた|撃った}。"
+LOOKUP_ROWS_FOUND: "{0}行見つかりました。"
+LOOKUP_SEARCHING: "検索中。しばらくお待ちください..."
+LOOKUP_STORAGE: "{0}が{2} {1}を{預け入れ|引き出し}た。"
+LOOKUP_TIME: "{0}前"
+LOOKUP_USERNAME: "{0}が{1}としてログイン"
+MAXIMUM_RADIUS: "最大半径の{0}中で{検索|ロールバック|復元}をします。"
 MISSING_ACTION_USER: "そのアクションを使用するには、ユーザーを指定してください。"
-MISSING_LOOKUP_TIME: "時間を指定してください({ルックアップ|ロールバック|復元})"
+MISSING_LOOKUP_TIME: "時間を指定してください({検索|ロールバック|復元})"
 MISSING_LOOKUP_USER: "ユーザーを指定するか、 {ブロック|半径}を指定してください。"
 MISSING_PARAMETERS: "\"{0}\"のパラメーターは存在していません。"
-MISSING_ROLLBACK_RADIUS: "指定しませんでした {ロールバック|復元} 半径。"
-MISSING_ROLLBACK_USER: "指定しませんでした {ロールバック|復元} ユーザー。"
-MYSQL_UNAVAILABLE: "接続できません MySQL サーバ。"
+MISSING_ROLLBACK_RADIUS: "{ロールバック|復元}対象の半径を指定しませんでした。"
+MISSING_ROLLBACK_USER: "{ロールバック|復元}対象のユーザーを指定しませんでした。"
+MYSQL_UNAVAILABLE: "MySQLサーバーに接続できません"
 NO_DATA: "{0}でデータが見つかりません。"
-NO_DATA_LOCATION: "番号 {データ|トランザクション|相互作用|メッセージ} 見つかった。"
+NO_DATA_LOCATION: "{データ|トランザクション|相互作用|メッセージ}は見つかりませんでした。"
 NO_PERMISSION: "あなたにはそれをする許可がありません。"
-NO_RESULTS: "結果が見つかりません。"
-NO_RESULTS_PAGE: "番号 {結果|データ} そのページで見つかりました。"
-NO_ROLLBACK: "番号 {保留中|前へ} ロールバック/復元が見つかりました。"
+NO_RESULTS: "結果が見つかりませんでした。"
+NO_RESULTS_PAGE: "このページには{結果|データ}は見つかりませんでした。"
+NO_ROLLBACK: "{保留中の|進んでいる}ロールバック/復元は見つかりませんでした。"
 PATCH_INTERRUPTED: "アップグレードが中断されました。再起動時に再試行します。"
-PATCH_OUTDATED_1: "古いデータベースをアップグレードできません {0}。"
-PATCH_OUTDATED_2: "CoreProtectがサポートされているバージョンでアップグレードしてください。"
-PATCH_PROCESSING: "新しいデータを処理しています。お待ちください..."
-PATCH_SKIP_UPDATE: "スキップ {テーブル|インデックス} {更新|作成|削除} の上 {0}。"
-PATCH_STARTED: "実行する {0} アップグレード。お待ちください..."
-PATCH_SUCCESS: "正常にアップグレードされました {0}。"
-PATCH_UPGRADING: "データベースのアップグレードが進行中です。お待ちください..."
-PLEASE_SELECT: "選んでください: \"{0}\" また \"{1}\" 。"
+PATCH_OUTDATED_1: "{0}は古いデータベースをアップグレードできません。"
+PATCH_OUTDATED_2: "CoreProtect がサポートされているバージョンにアップグレードしてください。"
+PATCH_PROCESSING: "新しいデータを処理しています。しばらくお待ちください..."
+PATCH_SKIP_UPDATE: "{0}の{テーブル|インデックス}の{更新|作成|削除}をスキップします。"
+PATCH_STARTED: "{0} のアップグレードを実行します。しばらくお待ちください..."
+PATCH_SUCCESS: "{0} は正常にアップグレードされました。"
+PATCH_UPGRADING: "データベースのアップグレードが進行中です。しばらくお待ちください..."
+PLEASE_SELECT: "選んでください: \"{0}\" または \"{1}\" 。"
 PREVIEW_CANCELLED: "プレビューがキャンセルされました。"
 PREVIEW_CANCELLING: "プレビューをキャンセルしています..."
 PREVIEW_IN_GAME: "ゲーム内でのみロールバックをプレビューできます。"
-PREVIEW_TRANSACTION: "プレビューできません {コンテナ|在庫} トランザクション。"
+PREVIEW_TRANSACTION: "{コンテナ|在庫}の入れ替えがプレビューできません"
 PURGE_ABORTED: "パージに失敗しました。データベースが破損している可能性があります。"
 PURGE_ERROR: "{0} のデータを処理できません"
-PURGE_FAILED: "パージに失敗しました。後でもう一度やり直してください。"
-PURGE_IN_PROGRESS: "進行中のパージ。後でもう一度やり直してください。"
-PURGE_MINIMUM_TIME: "分離できるのは、より古いデータのみです。({0}{日|時間})"
-PURGE_NOTICE_1: "これには時間がかかる場合がありますのでご注意ください。"
+PURGE_FAILED: "パージに失敗しました。もう一度やり直してください。"
+PURGE_IN_PROGRESS: "パージは進行中です。もう一度やり直してください。"
+PURGE_MINIMUM_TIME: "パージできるのは、より古いデータのみです。({0}{日|時間})"
```

```suggestion
PURGE_MINIMUM_TIME: "パージできるのは、({0}{日|時間})より古いデータのみです。"
```

`パージできるのは、1時間より古いデータのみです。` 
みたいになるはず（そうすると期間を表すので日間のほうが良いかも)

`Only data older than 1 hour can be purged. ` 
should look something like this (then it might be better to use days since it represents a period of time).

---

そうするとこっちも
Then we'll get this one too.

https://github.com/PlayPro/CoreProtect/pull/672/files#diff-ee8bbe2e987b34d6f6b79d09f1a5317d2fab8a46e0d079e0e56a3471d6342991R185-R189

日間、週間、分間などの期間時間にしたほうが良いかもしれない
Might be better to make it time period time, such as days, weeks, minutes, etc.